pom.xml

```diff
@@ -58,12 +58,48 @@
 		    <artifactId>flyway-core</artifactId>
 		    <version>5.1.4</version>
 		</dependency>
-
-        <dependency>
+		 <dependency>
         	<groupId>org.projectlombok</groupId>
         	<artifactId>lombok</artifactId>
-        	<version>1.18.2</version>
+        	 <version>1.18.2</version>
         </dependency>
+        <dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-security</artifactId>
+		</dependency>
```

Comment:
This looks a bit messy. Could you please reformat the file so that it is formatted nicely, with closing and opening elements with the same indent?